cmd/occollector/app/builder/testdata/processor_config.yaml

```diff
@@ -1,18 +1,18 @@
-processors:
+ queued-exporters:
```

Comment:
Should we rename the file to match the `processors` -> `queued-exporters` rename?